src/components/PrimaryPanes/Outline.css

```diff
@@ -39,6 +39,10 @@
   list-style: none;
 }
 
+.outline-list__class-list > .outline-list__element {
+  padding-left: 2rem;
```

Comment:
Probably `padding-inline-start` here to be consistent with the style for `.outline-list > .outline-list__element`.

Reply:
Yep, let's move to `padding-inline-start`